### PR TITLE
Optimize database

### PR DESCRIPTION
**Pull request by schoetju**
Enables the user to do a database optimization from NextcloudPi UI like mentioned in 
https://github.com/nextcloud/nextcloudpi/issues/989

### CODE REVIEW

**Comment by theCalcaholic** [comment]
Thanks for the PR! I'll consider it vor v1.53.3

File: bin/ncp/TOOLS/db-optimize.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Optimize MariaDB
+#
+
+configure()
+{
+  echo -ne "begin of MariaDB Optimization..."
+  mariadb-check --optimize --all-databases >/dev/null
```
Comment (theCalcaholic) [inline bin/ncp/TOOLS/db-optimize.sh:9]: This command doesn't work for me without setting the user and also I think it's good to show users the output. Can you change it to:

```diff
-   mariadb-check --optimize --all-databases >/dev/null
+   mariadb-check -u root --optimize --all-databases
```

**Review by theCalcaholic — CHANGES_REQUESTED** [review]
Other than the suggested change, it looks good :)

File: bin/ncp/TOOLS/db-optimize.sh (hunk 1 above)
Comment (schoetju) [reply bin/ncp/TOOLS/db-optimize.sh]: Makes sense. I've changed file accordingly.

Comment (theCalcaholic) [reply bin/ncp/TOOLS/db-optimize.sh]: Thank you, looks good :)

**Review by theCalcaholic — APPROVED** [review]

**Pull request merged**